docs/HowTo/installation/run-source.md

```diff
@@ -74,4 +74,4 @@ the source.
 ## Next Steps
 
 Once you have finished verifying your installation, we recommend you follow the documentation
-in [configure-klaw-wizard](configure-klaw-wizard.md).
+in [configure-klaw-wizard](configure-Klaw-wizard.md).
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?